AttributeError raised when calling configure_mpl_tex

I am seeing the following error when running the gwdetchar test suite:

```python
$ python -m pytest gwdetchar -k test_configure_mpl_tex
======================================================================= test session starts =======================================================================
platform linux -- Python 3.9.2, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/duncan/git/gwdetchar-fork, configfile: setup.cfg
plugins: forked-1.3.0, cov-2.11.1, xdist-2.2.1, requests-mock-1.8.0
collected 145 items / 144 deselected / 1 selected

gwdetchar/lasso/tests/test_plot.py F                                                                                                                        [100%]

============================================================================ FAILURES =============================================================================
_____________________________________________________________________ test_configure_mpl_tex ______________________________________________________________________

    def test_configure_mpl_tex():
>       plot.configure_mpl_tex()

gwdetchar/lasso/tests/test_plot.py:45:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    def configure_mpl_tex():
        """Configure Matplotlib with LaTeX when using multiprocessing
        """
        import matplotlib
        matplotlib.use('agg')

        mpldir = tempfile.mkdtemp()
        atexit.register(shutil.rmtree, mpldir)
        umask = os.umask(0)
        os.umask(umask)
        os.chmod(mpldir, 0o777 & ~umask)
        os.environ['HOME'] = mpldir
        os.environ['MPLCONFIGDIR'] = mpldir

>       class TexManager(matplotlib.texmanager.TexManager):
E       AttributeError: module 'matplotlib' has no attribute 'texmanager'

gwdetchar/lasso/plot.py:50: AttributeError
======================================================================== warnings summary =========================================================================
../../opt/miniconda3/envs/py39/lib/python3.9/site-packages/pycondor/job.py:4
  /home/duncan/opt/miniconda3/envs/py39/lib/python3.9/site-packages/pycondor/job.py:4: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    from collections import namedtuple, Iterable

../../opt/miniconda3/envs/py39/lib/python3.9/site-packages/gwpy/signal/spectral/_registry.py:78
  /home/duncan/opt/miniconda3/envs/py39/lib/python3.9/site-packages/gwpy/signal/spectral/_registry.py:78: DeprecationWarning: the default spectral averaging method is currently 'welch' (mean averages of overlapping periodograms), but this will change to 'median' as of gwpy-2.1.0
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================================================= 1 failed, 144 deselected, 2 warnings in 21.50s ==========================================================
```

This can be simplified to the following:

```pytb
$ python -c "import matplotlib; print(matplotlib.texmanager)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'matplotlib' has no attribute 'texmanager'
```

This seems to be the case with all modern versions of matplotlib, so I have no idea how this hasn't been seen before. My only guess is that this has been a lucky coincidence with some sort of race condition relating to someone else importing `matplotlib.texmanager` first.

I'll post a patch.

--- a/gwdetchar/lasso/plot.py
+++ b/gwdetchar/lasso/plot.py
@@ -39,6 +39,8 @@ def configure_mpl_tex():
     import matplotlib
     matplotlib.use('agg')
 
+    from matplotlib import texmanager
+
     mpldir = tempfile.mkdtemp()
     atexit.register(shutil.rmtree, mpldir)
     umask = os.umask(0)
@@ -47,10 +49,10 @@ def configure_mpl_tex():
     os.environ['HOME'] = mpldir
     os.environ['MPLCONFIGDIR'] = mpldir
 
-    class TexManager(matplotlib.texmanager.TexManager):
+    class TexManager(texmanager.TexManager):
         texcache = os.path.join(mpldir, 'tex.cache')
 
-    matplotlib.texmanager.TexManager = TexManager
+    texmanager.TexManager = TexManager
     matplotlib.rcParams['ps.useafm'] = True
     matplotlib.rcParams['pdf.use14corefonts'] = True
     matplotlib.rcParams['text.usetex'] = True

